greek_verbs/src/anki_flashcard.py

```diff
@@ -27,17 +27,20 @@ def conjugation_to_html(verb: str, conj: dict) -> str:
     return html_template
 
 
-def format_example_usages(conj: dict, example_usages: dict) -> str:
+def format_example_usages(conj: dict, example_usages: dict, num_examples) -> str:
     """
     Extract example usages from the conjugation and format them for
     an Anki flashcard. The input is a dictionary with key:value pairs
     of Greek:English for each example usage.
     """
     if len(example_usages):
+
         template = '{greek}<br><em>{english}</em>'
 
         usages = []
-        for greek, english in example_usages.items():
+
+        for i, (greek, english) in enumerate(example_usages.items()):
```

Comment:
`i` will start 0, so when `i == num_examples` there will be `num_examples+1` usages appended

Reply:
no there won't be. I tested it